.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Add Tags to Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
```

Comment:
Let's trigger on:

```
  release:
    types: [published]
```

Reply:
https://help.github.com/en/actions/reference/events-that-trigger-workflows#release-event-release